### DISCUSSION TITLE
notes/daphne-liu/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#40] [Announcements]
# notes/daphne-liu/

Notes about the cascadiajs 2022 conference

https://cascadiajs.dendron.so/notes/saimon-sharif/

**Comment by vim-usds** [comment]
Slides here - https://docs.google.com/presentation/d/1f2aMU0tW67sW15dOVwZzcDyae4ecegtwYerq_KiTiNM/edit#slide=id.ge2600e9d84_0_0

**Comment by vim-usds** [comment]
slides here - https://www.canva.com/design/DAFD-lzA61U/v517F9SjB4mJz2laYc8U-Q/view#1
